include/dcmqi/MultiframeConverter.h

```diff
@@ -35,15 +35,21 @@
 
 using namespace std;
 
+// common type definitions
 typedef short ShortPixelType;
 typedef itk::Image<ShortPixelType, 3> ShortImageType;
 typedef itk::ImageFileReader<ShortImageType> ShortReaderType;
 
 namespace dcmqi {
 
-  class ConverterBase {
+  class MultiframeConverter {
```

Comment:
@michaelonken is there a preference in DCMTK for Multiframe vs. MultiFrame?

I see several examples of both currently...

(cd DCMTK; git grep -i multiframe) | less